ecosystem/python/sdk/README.md

```diff
@@ -42,7 +52,7 @@ Currently the team is moving forward with pure python, but leaves the following
 ```bash
 npm install  -g @apidevtools/swagger-cli
 swagger-cli bundle --dereference ../../../api/doc/v0/openapi.yaml  -t yaml > openapi.yaml
-python -m openapi_python_client generate --path openapi.yaml
+python3 -m openapi_python_client generate --path openapi.yaml
```

Comment:
Can't comment below.  Are you supposed to be committing pycache?

Reply:
Hmm... I'll revisit the stack... they must have snuck in and then I removed them at the end :)